.travis.yml

```diff
@@ -1,20 +1,20 @@
 language: objective-c
 xcode_workspace: Example/TraceLog.xcworkspace
 xcode_scheme: TraceLog-Example
```

Comment:
@phchang can you switch out `xcode_workspace:` and `xcode_scheme:` here for `os: osx` as it is in GeoFeatures 1?  These are redundant and don't need to be here, just more for us to maintain and keep in sync.